exercise_14.js

```diff
@@ -0,0 +1,43 @@
+/*
+Make multiple functions that will return the sum, difference, modulus, product, quotient, and the exponent respectively.
+
+Please use the following function names:
+
+addition = add
+
+multiply = multiply
+
+division = divide (both integer and float divisions are accepted)
+
+modulus = mod
+
+exponential = exponent
+
+subtraction = subt
+
+Note: All math operations will be: a (operation) b
+ */
+
+function add(a,b){
+    return a + b;
+}
+
+function divide(a,b){
+    return a / b;
+}
+
+function multiply(a,b){
+    return  a * b;
+}
+
+function mod(a,b){
+    return a % b;
+}
+
+function exponent(a,b){
+    return a**b;
+}
+
+function subt(a,b){
+    return  a -b;
+}
```

Comment:
please be consistent in formatting code. If you add a space before and after the sign, make sure that you do it everywhere